src/Features/Core/Portable/RemoveUnusedParametersAndValues/AbstractRemoveUnusedValuesCodeFixProvider.cs

```diff
@@ -391,7 +391,8 @@ protected sealed override async Task FixAllAsync(Document document, ImmutableArr
             // at declaration, then we also add a new variable declaration statement without initializer for this local.
 
             var nodeReplacementMap = PooledDictionary<SyntaxNode, SyntaxNode>.GetInstance();
-            var nodesToRemove = PooledHashSet<SyntaxNode>.GetInstance();
+            var nodesToRemove = PooledDictionary<SyntaxNode, bool>.GetInstance();
```

Comment:
this variable shoudl be renamd to better indicate what the value indicates.  i.e. nodeToXXXMapping.